src/IndexedSectionList.tsx

```diff
@@ -49,10 +49,21 @@ export default React.forwardRef(
       scrollEfficiency,
       wrapperStyle,
       indexWrapperStyle,
+      indexTextStyle,
+      sectionTitleStyle,
+      hideSectionDivider,
+      sectionDividerStyle,
       getSectionProps,
       uniqueKey = 'key' as UniqueKey,
       titleKey = 'title' as TitleKey,
-      renderSectionHeader = ({ section: { title } }) => <SectionHeader title={title} />,
+      renderSectionHeader = ({ section: { title } }) => (
```

Comment:
Wouldn't it be simpler to just use your own `renderSectionHeader` in order to customize your SectionHeader, instead of adding 3 more props? That's what the `renderSectionHeader` is prop for.

Reply:
You are right. Updated PR.